packages/search/README.md

```diff
@@ -1,88 +1,147 @@
+### Options
+
+#### Sizes
+
+Search fields come in four different sizes for mobile and desktop platform scales: small, medium, large, and extra-large. The medium size is the default and most frequently used option.
 
 <sp-tabs selected="m" auto label="Size Attribute Options">
 <sp-tab value="s">Small</sp-tab>
 <sp-tab-panel value="s">
 
-```html
+```html demo
 <sp-search size="s"></sp-search>
-<sp-search size="s" disabled></sp-search>
 ```
 
 </sp-tab-panel>
 <sp-tab value="m">Medium</sp-tab>
 <sp-tab-panel value="m">
 
-```html
+```html demo
 <sp-search></sp-search>
-<sp-search disabled></sp-search>
 ```
 
 </sp-tab-panel>
 <sp-tab value="l">Large</sp-tab>
 <sp-tab-panel value="l">
 
-```html
+```html demo
 <sp-search size="l"></sp-search>
-<sp-search size="l" disabled></sp-search>
 ```
 
 </sp-tab-panel>
 <sp-tab value="xl">Extra Large</sp-tab>
 <sp-tab-panel value="xl">
 
-```html
+```html demo
 <sp-search size="xl"></sp-search>
-<sp-search size="xl" disabled></sp-search>
 ```
 
 </sp-tab-panel>
 </sp-tabs>
 
-## Variants
+#### Label and icon
+
+A search field should include a label. In the default state before a search term is input, the label is in regular body text style to meet contrast ratios and to show that this is a field label, not placeholder text. The search icon is set using the `icon magnifier icon-workflow icon-search` classes in the component.
+
+```html demo
+<sp-search label="This is a label"></sp-search>
+```
+
+#### Placeholder
+
+A placeholder is a hint to the user about what to input in the search field. It is displayed when the search field is empty and is removed when the user starts typing.
+
+```html demo
+<sp-search placeholder="Enter a search term"></sp-search>
+```
+
+#### Help text
 
-### Quiet
+A search field can have help text below the field to give extra context or instruction about what a user should input. The description communicates a hint or helpful information, such as a search's scope.
 
-```html
-<sp-search quiet></sp-search>
-<sp-search quiet disabled></sp-search>
+```html demo
+<sp-search>
+    <sp-help-text slot="help-text">Enter a search term.</sp-help-text>
+    <sp-help-text slot="negative-help-text">Invalid search term.</sp-help-text>
+</sp-search>
 ```
 
-## Events
+### States
+
+#### Disabled
+
+A search field in a disabled state shows that a search option exists, but is not available in that circumstance. This can be used to maintain layout continuity and communicate that it may become available later.
+
+```html demo
+<sp-search disabled></sp-search>
+```
+
+### Events
+
+The `submit` event fires when the `<sp-search>` is submitted. This is a non-`composed` event inline with what you would expect a [`<form />`](https://developer.mozilla.org/en-US/docs/Web/API/HTMLFormElement/submit_event) to fire. If you choose to prevent the default of this event, the default action for the underlying `<form />` element will also be prevented, which will allow you to handle the search action in javascript.
```

Comment:
```suggestion
The `submit` event fires when the `<sp-search>` is submitted. This is a non-`composed` event in line with what you would expect a [`<form />`](https://developer.mozilla.org/en-US/docs/Web/API/HTMLFormElement/submit_event) to fire. If you choose to prevent the default behavior of this event, the default action for the underlying `<form />` element will also be prevented, allowing you to handle the search action in JavaScript.
```

Also wondering if it should be `<form>` instead of `<form/>`, since `<form>` isn't self-closing?